### PR TITLE
Add installation and configuration info to README

### PR DESCRIPTION
**Pull request by mpeck**
resolves https://github.com/operable/cog/issues/930

### CODE REVIEW

**Comment by mpeck** [comment]
Should I duplicate this info in the docs, maybe under a section called "Configuring cogctl locally", or is it enough to just include it in the README here?

File: README.md [hunk 1]
```diff
@@ -2,36 +2,92 @@
...
+    mix local.hex --force
+    mix local.rebar --force
+
+### Compiling
+
+After getting all the prerequisites out of the way you can pull cogctl's deps and compile.
+
+    mix deps.get
+    mix deps.compile
+    mix escript
```
Comment (vanstee) [inline README.md]: I think you can actually just run `mix escript` here.

Comment (vanstee) [inline README.md]: I guess it might still be worth showing those other two commands, but it might not be necessary. Not a big deal either way.

Comment (mpeck) [inline README.md]: Yeah, you're right. I guess I could have looked in the mix file :)

**Pull request merged**